skrifa/src/outline/autohint/latin/edges.rs

```diff
@@ -223,16 +277,25 @@ pub(crate) fn compute_blue_edges(
     scale: &Scale,
     unscaled_blues: &[UnscaledBlue],
     blues: &[ScaledBlue],
+    group: ScriptGroup,
 ) {
-    if axis.dim != Axis::VERTICAL {
+    // For the default script group, we only handle vertical blues
+    if axis.dim != Axis::VERTICAL && group == ScriptGroup::Default {
         return;
     }
+    let axis_scale = if axis.dim == Axis::HORIZONTAL {
+        scale.x_scale
+    } else {
+        scale.y_scale
+    };
+    // Initial threshold
+    let initial_best_dest = fixed_mul(scale.units_per_em / 40, axis_scale).min(64 / 2);
     for edge in &mut axis.edges {
         let mut best_blue = None;
         let mut best_is_neutral = false;
```

Comment:
Optional. I don't have time to try it but finding the best values for an edge smells a lot like reduce. Is it possible it would make sense to write this in terms of reduce? - if it does but we don't want to do it now feel free to defer to an issue.

Reply:
Added to #1141. This seems possible but the interior code would need to be factored out into a function to make the change worthwhile.